etc/version.lock.json

```
@@ -345,7 +345,7 @@
     "version": 2
   },
   "1d72d014-e2ab-4707-b056-9b96abe7b511": {
-    "rule_name": "External IP Lookup fron Non-Browser Process",
+    "rule_name": "External IP Lookup from Non-Browser Process",
```

Nice catch, but we can revert this. The rule name is properly updated below, but updates to the version lock are only done at specific times, programmatically


```suggestion
    "rule_name": "External IP Lookup fron Non-Browser Process",
```